lte/gateway/c/sctpd/src/sctpd.cpp

```diff
@@ -75,11 +84,58 @@ int signalHandler(int* end, std::unique_ptr<Server>& server,
   return 0;
 }
 
+static magma::mconfig::SctpD load_sctpd_mconfig() {
+  magma::mconfig::SctpD mconfig;
+  if (!magma::load_service_mconfig_from_file(SCTPD_SERVICE, &mconfig)) {
+    mconfig.set_log_level(magma::orc8r::LogLevel::INFO);
+  }
+  return mconfig;
+}
+
+static magma::mconfig::SharedMconfig load_shared_mconfig() {
+  magma::mconfig::SharedMconfig mconfig;
+  magma::load_service_mconfig_from_file(SHARED_MCONFIG, &mconfig);
+  return mconfig;
+}
+
+static uint32_t get_log_verbosity(const YAML::Node& config,
+                                  magma::mconfig::SctpD mconfig) {
+  if (!config["log_level"].IsDefined()) {
+    return magma::get_log_verbosity_from_mconfig(mconfig.log_level());
+  }
+  std::string log_level = config["log_level"].as<std::string>();
+  if (log_level == "DEBUG") {
+    return MDEBUG;
+  } else if (log_level == "INFO") {
+    return MINFO;
+  } else if (log_level == "WARNING") {
+    return MWARNING;
+  } else if (log_level == "ERROR") {
+    return MERROR;
+  } else if (log_level == "FATAL") {
+    return MFATAL;
+  } else {
+    MLOG(MINFO) << "Invalid log level in config: "
+                << config["log_level"].as<std::string>();
+    return MINFO;
```

Comment:
This conversion is used in a lot of services and could be extracted somewhere (maybe as a function in ServiceConfigLoader?).